util/csrankings.py

```diff
@@ -15,8 +15,17 @@
 import operator
 from builtins import str
 
-# Papers must be at least 6 pages long to count.
-pageCountThreshold = 6
+# Papers must be at least 6 pages long to count, unless overridden
+# in pageCountThresholdDict.
+defaultPageCountThreshold = 6
+
+# Optional override of page count threshold for specific venues.
+pageCountThresholdDict = {
+    'FOCS': 4,
```

Comment:
I am not going to adopt these changes as part of this PR. Please delete the dict entries.